connect-tests/Cargo.toml

```diff
@@ -2,7 +2,6 @@
 name = "rustls-connect-tests"
 version = "0.0.1"
 edition = "2021"
-rust-version = "1.60"
```

Comment:
Why are these `rust-version` directives falling away here, maybe a candidate for a separate commit? The [workspace docs](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-package-table) read to me like we need a `[workspace.package]` in the top-level `Cargo.toml` for this to be inherited, is that incorrect?

Reply:
Pulled the MSRV changes out into #1516.